src/io/observe.cc

```diff
@@ -141,110 +215,51 @@ void ObservePoint::find_observe_point(Mesh &mesh) {
     const BIHTree &bih_tree=mesh.get_bih_tree();
     vector<unsigned int> candidate_list, process_list;
     std::unordered_set<unsigned int> closed_elements(1023);
+    std::priority_queue< ObservePointData, std::vector<ObservePointData>, CompareByDist > candidate_queue;
 
     // search for the initial element
     bih_tree.find_point(input_point_, candidate_list);
```

Comment:
Add parameter to return leaf elements without checking that point is contained in their bboxes